mkdocs.yml

```diff
@@ -22,6 +22,7 @@ pages:
   - 'Databases':
       - 'MongoDB': 'databases/mongo.md'
       - 'SQLite': 'databases/sqlite.md'
+      - 'Rocket.Chat': 'connectors/rocketchat.md'
```

Comment:
This should be in connectors not databases 😄

Reply:
Ooops thats right haha